omod-common/pom.xml

```diff
@@ -33,7 +33,7 @@
 		<dependency>
 			<groupId>io.swagger</groupId>
 			<artifactId>swagger-core</artifactId>
-			<version>1.5.13</version>
+			<version>1.6.1</version>
```

Comment:
wel done @achilep .

Is this the latest version really ??
https://mvnrepository.com/artifact/io.swagger/swagger-core

Reply:
thanks @mozzy11  i will use the 1.6.2 version